pyth_observer/prices.py

```diff
@@ -113,6 +113,14 @@ def update_slot(self, slot: Optional[int]) -> None:
         if self.last_updated_slot is None or slot > self.last_updated_slot:
             self.last_updated_slot = slot
 
+    def update_coingecko_price(self, coingecko_price: Optional[float]) -> None:
```

Comment:
this seems like it's not related to the rest of the change (?) if so, please remove from this PR.